src/model/mod.rs

```diff
@@ -421,70 +347,26 @@ mod test {
     }
 
     #[test]
-    fn test_provider_config() {
-        let manifest =
-            deserialize_yaml("./oam/provider_config.yaml").expect("Should be able to parse");
-        let props = match &manifest.spec.components[0].properties {
-            Properties::Capability { properties } => properties,
-            _ => panic!("Should have found capability component"),
-        };
-
-        match props.config.as_ref().expect("Config should have been set") {
-            CapabilityConfig::Opaque(data) => assert_eq!(data, "{\"raw\": \"json\", \"data\": {}}"),
-            _ => panic!("Should have found opaque config"),
-        }
+    fn test_config() {
+        let manifest = deserialize_yaml("./oam/config.yaml").expect("Should be able to parse");
+        // TODO: test component
```

Comment:
Issue here? this might be a great first issue/something for someone else to pick up?

Reply:
🤔 Oh that might be left over. If not will file little issue :)